etna/models/sarimax.py

```diff
@@ -534,7 +534,7 @@ def forecast(
 
         result_list = list()
         for segment in self._segments:
-            model = self._models[segment]
+            model = self._models[segment]  # type: ignore
 
             segment_predict = self._forecast_one_segment(model, segment, ts, prediction_interval, quantiles)
```

Comment:
Does it mean, that after this PR SARIMAX no longer support prediction intervals until #507 is done?

Reply:
No it does't, Prophet and Sarimax have their own fit/forrecast